qa-tests-backend/src/test/groovy/SACTest.groovy

```diff
@@ -603,63 +603,59 @@ class SACTest extends BaseSpecification {
     }
 
     def "Verify that SAC has the same effect as query restriction for network flows"() {
```

Comment:
Instead of retrying nearly whole test you can use `@Retry` on the top of the test:
https://spockframework.org/spock/javadoc/2.0/spock/lang/Retry.html